README.md

```diff
@@ -78,16 +78,18 @@ az account show
 
 ### 2. Use inside a Development Container
 
+Using a development container will mean that all of the required software is installed for you. This will utilise WSL.
```

Comment:
```suggestion
Using a development container will mean that all of the required software is installed for you. This will utilise WSL. For more information about development containers visit (https://containers.dev/)[containers.dev]
```

Reply:
Don't accept my suggested commit above. I have my brackets the wrong way around!!!